src/app/parcels/add/[clientId]/page.tsx

```diff
@@ -1,101 +1,24 @@
-            break;
-        case "packingSlotsFetchFailed":
-            errorMessage = "Failed to fetch packing slots data.";
-            break;
-    }
-    return `${errorMessage} Log ID: ${error.logId}`;
-};
-
-const AddParcels = ({ params }: AddParcelParameters): React.ReactElement => {
-    const [deliveryPrimaryKey, setDeliveryPrimaryKey] = useState<string | null>(null);
-    const [collectionCentresLabelsAndValues, setCollectionCentresLabelsAndValues] =
-        useState<CollectionCentresLabelsAndValues | null>(null);
-    const [packingSlotsLabelsAndValues, setPackingSlotsLabelsAndValues] =
-        useState<PackingSlotsLabelsAndValues | null>(null);
-    const [error, setError] = useState<FetchCollectionCentresError | PackingSlotsError | null>(
-        null
-    );
-    const [isLoading, setIsLoading] = useState(true);
-
-    useEffect(() => {
-        (async () => {
-            setIsLoading(true);
-            const supabase = getLocalSupabaseClient();
-
-            const { data: collectionCentresData, error: collectionCentresError } =
-                await getCollectionCentresInfo(supabase);
-            if (collectionCentresError) {
-                setError(collectionCentresError);
-                setIsLoading(false);
-                return;
-            }
-            setCollectionCentresLabelsAndValues(
-                collectionCentresData.collectionCentresLabelsAndValues
-            );
-            setDeliveryPrimaryKey(collectionCentresData.deliveryPrimaryKey);
-
-            const { data: packingSlotsData, error: packingSlotsError } =
-                await fetchPackingSlotsInfo(supabase);
-            if (packingSlotsError) {
-                setError(packingSlotsError);
-                setIsLoading(false);
-                return;
-            }
-            setPackingSlotsLabelsAndValues(packingSlotsData);
-            setIsLoading(false);
-        })();
-    }, []);
-
+const AddParcel = async ({ params }: AddParcelParameters): Promise<React.ReactElement> => {
     return (
         <main>
-            <Title>Parcel Form</Title>
-            {isLoading ? (
-                <></>
-            ) : error ? (
-                <ErrorSecondaryText>{getErrorMessage(error)}</ErrorSecondaryText>
-            ) : (
-                <ParcelForm
-                    initialFields={initialParcelFields}
-                    initialFormErrors={initialParcelFormErrors}
-                    clientId={params.clientId}
-                    editMode={false}
-                    deliveryPrimaryKey={deliveryPrimaryKey!}
-                    collectionCentresLabelsAndValues={collectionCentresLabelsAndValues!}
-                    packingSlotsLabelsAndValues={packingSlotsLabelsAndValues!}
-                />
-            )}
+            <AddParcelForm clientId={params.clientId} />
         </main>
     );
 };
 
-export default AddParcels;
+export const metadata: Metadata = {
+    title: "Edit Parcels",
```

Comment:
Was this meant to be Add Parcels? What was the original title?

Reply:
Thank you for catching, it should be Add Parcel